### PR TITLE
Features spec - Document 'mounts' property

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Reported in https://github.com/devcontainers/spec/issues/173

### CODE REVIEW

File: docs/specs/devcontainer-features.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The properties of the file are as follows:
 | `installsAfter` | array | Array of ID's of Features (omitting a version tag) that should execute before this one. Allows control for Feature authors on soft dependencies between different Features. |
 | `legacyIds` | array | Array of old IDs used to publish this Feature. The property is useful for renaming a currently published Feature within a single namespace. |
 | `deprecated` | boolean | Indicates that the Feature is deprecated, and will not receive any further updates/support. This property is intended to be used by the supporting tools for highlighting Feature deprecation. |
+| `mounts` | string or object | Defaults to unset. Cross-orchestrator way to add additional mounts to a container. Each value is a string that accepts the same values as the [Docker CLI `--mount` flag](https://docs.docker.com/engine/reference/commandline/run/#add-bind-mounts-or-volumes-using-the---mount-flag). The Pre-defined [devcontainerId](./devcontainerjson-reference.md/#variables-in-devcontainerjson) variable may be referenced in the value. For example:<br />`"mounts": [{ "source": "dind-var-lib-docker", "target": "/var/lib/docker", "type": "volume" }]` |
```
Comment (chrmarti) [inline docs/specs/devcontainer-features.md:50]: We only support the object form. (devcontainer.json also supports the string representation, but that is slightly more Docker-specific).

**Review by chrmarti — CHANGES_REQUESTED** [review]
One nit, thanks.

File: docs/specs/devcontainer-features.md (hunk 1 above)
Comment (samruddhikhandale) [reply docs/specs/devcontainer-features.md]: I see. Updated thanks!

**Review by chrmarti — CHANGES_REQUESTED** [review]

File: docs/specs/devcontainer-features.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The properties of the file are as follows:
...
-| `mounts` | string or object | Defaults to unset. Cross-orchestrator way to add additional mounts to a container. Each value is a string that accepts the same values as the [Docker CLI `--mount` flag](https://docs.docker.com/engine/reference/commandline/run/#add-bind-mounts-or-volumes-using-the---mount-flag). The Pre-defined [devcontainerId](./devcontainerjson-reference.md/#variables-in-devcontainerjson) variable may be referenced in the value. For example:<br />`"mounts": [{ "source": "dind-var-lib-docker", "target": "/var/lib/docker", "type": "volume" }]` |
+| `mounts` | object | Defaults to unset. Cross-orchestrator way to add additional mounts to a container. Each value is a string that accepts the same values as the [Docker CLI `--mount` flag](https://docs.docker.com/engine/reference/commandline/run/#add-bind-mounts-or-volumes-using-the---mount-flag). The Pre-defined [devcontainerId](./devcontainerjson-reference.md/#variables-in-devcontainerjson) variable may be referenced in the value. For example:<br />`"mounts": [{ "source": "dind-var-lib-docker", "target": "/var/lib/docker", "type": "volume" }]` |
```
[the remark this answers is not in the document]
Comment (samruddhikhandale) [reply docs/specs/devcontainer-features.md]: Oops 😅 updated

**Review by joshspicer — APPROVED** [review]

**Pull request merged**